### ISSUE TITLE
Omensight (455820)

### ISSUE THREAD

**Issue by lieff** [ValveSoftware/Proton#1106]
Distro: Ubuntu 18.04
GPU: GTX 980, Nvidia proprietary driver 390.48
Resolution: 2560x1440
RAM: 16GB
Proton 3.7-5 Beta

First witch dialog text stuck on screen during cut-scenes. In-game part works perfectly.

**Comment by liberodark** [comment]
I think it's fixed in 3.16 Beta have try this game and is great

**Comment by alosarjos** [comment]
Cutscenes are not showing for me. Performance I think could be better.

**Comment by kisak-valve** [comment]
Omensight: Definitive Edition Crash on start up and crash the whole Steam Client

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8343.
@daishishi posted on 2024-12-25T16:07:17:

# Compatibility Report
- Omensight: Definitive Edition
- Steam AppID of the game: 455820

## System Information
- GPU: Radeon RX 6570XT
- Video driver version: Mesa 24.0.3-1pop1~1711635559~22.04~7a9f319
- Kernel version: 6.9.3-76060903-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/daishishi/455a9447a5580efb1b531d3f118767d4
- Proton version: Proton Hotfix / Proton Experimental / Proton 8 / Proton 7 / Proton 6

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [x ] that I have checked whether there are updates for my system available.

 
[steam-455820.log](https://github.com/user-attachments/files/18246605/steam-455820.log)
[Dumps.zip](https://github.com/user-attachments/files/18246607/Dumps.zip) -> Here are various dump files also.


## Symptoms  
The game crashes immediately when click Run, and also the whole steam client closes with it.
Weirdly I can run the game executable directly using Lutris, and the wine **ge-proton 8**, and it run without a problem with fmv.


## Reproduction
Try to run the game.

**Comment by kisak-valve** [comment]
Hello @daishishi, as an extra data point, please note what update channel you're using with the Steam client (the client itself) and if you're using the Steam client beta, if the game behaves differently when opting out off that.

**Comment by daishishi** [comment]
Hello, thanks for moving the thread. I reckon making a new thread at first, as this one wasn't about a hard crash, and because on protondb this game apparently run flawless on steamdeck. (?)

I tried opting out of the **beta Stem client**, reinstalled it and tried running the game again. The game still crashes on start up, but the client don't close with it now. (Should I report this client crash in the beta client issue tracker?)
Here a new log file with the **non-beta client**: 
[steam-455820.log](https://github.com/user-attachments/files/18246701/steam-455820.log)

Thanks

**Comment by daishishi** [comment]
TLDR: The game works now after deleting the Wine Prefix folder, and launching the game with the stable Steam Client. **NO FMV PLAYBACK THOUGH**

An important note: After trying many things, before opening this issue, as last resort I used **ProtonTricks** to install the _Microsoft Media Foundation_ .dll. Which didn't solve the issue.
So after reverting the client back to the stable release, I tried cleaning the Wine Prefix to see, and now the game works. Although all FMV won't play: The 'Spearhead Studio' and 'Unreal' at the start, the intro when clicking new game, or even the ones playing during the tutorial popups.

**Comment by simifor** [comment]
@daishishi it looks like this particular game requires h264 for video playback, and on top of that there's a regression on proton 9 which makes the ingame video (at least the one that plays when you start a new game) with distortions.

Both of these should be taken care of in the upcoming days, but in the meantime, you can manually enable h264 support on your device by running the following command in your terminal: `steam steam://unlockh264/` and remember to switch proton versions to experimental.
